app/src/main/java/org/p2p/wallet/auth/repository/Country.kt

```diff
@@ -0,0 +1,9 @@
+package org.p2p.wallet.auth.repository
+
+class Country(
```

Comment:
maybe WorldCountry? Or GlobalCountry? Country looks very simple..

Reply:
Country is ok